Developer-Guide--Set-Up-With-Docker.md

```diff
@@ -179,3 +179,31 @@ If the `rails db:autotest_run` fails, you can still run the tests manually in yo
 5. Go to the Test Results tab and click on "Run Tests".
 6. Wait a minute, and then refresh the page. Go back to the Test Results tab. You should see that two tests have been run, and that both have passed.
 7. Repeat for the other assignments that you want to run tests for.
+
+### Setting up action mailer
+
+If you plan on doing work that involves sending/recieving emails from MarkUs, you will need to configure action mailer. To get you started quickly, here we will talk specifically about configuring action mailer with a gmail account using the `:smtp` delivery method.
```

Comment:
Say "ActionMailer", not "action mailer". Also please add a link to the ActionMailer guide for "configure ActionMailer".